build/grunt.js

```diff
@@ -85,19 +84,6 @@ module.exports = function(grunt) {
     },
     dist: {},
     watch: {
-      dist: {
-        files: [
-          'build/temp/video.js',
-          'build/temp/alt/video.novtt.js',
-          'build/temp/video-js.css',
```

Comment:
We no longer need this since `rollup` handles the the js files and `node-sass` will handle the css